common/src/main/java/dev/ftb/mods/ftblibrary/api/sidebar/SidebarButtonCreatedEvent.java

```diff
@@ -0,0 +1,23 @@
+package dev.ftb.mods.ftblibrary.api.sidebar;
+
+import dev.architectury.event.Event;
+import dev.architectury.event.EventFactory;
+import dev.ftb.mods.ftblibrary.sidebar.RegisteredSidebarButton;
+
+import java.util.function.Consumer;
+
+// TODO currently broken for neoforge, uncomment when there's a fix in architectury
+//@ForgeEvent
```

Comment:
This can probably just go; Daniel has indicated that this annotation will never be supported on Neoforge.